README.md

```diff
@@ -1,2 +1,45 @@
-# Recommended Equipment
-Recommends the equipment that you should use for different bosses and activities in the game.
\ No newline at end of file
+# Recommended Equipment OSRS Plugin
+> This plugin enhances your OSRS gameplay by seamlessly integrating
+> Equipment recommendations from the wiki directly into your RuneLite
+> client, and then filtering your bank. 
+> 
+> Streamline your gearing process for bossing and mini-games by
+> leveraging data directly from the OSRS Wiki without leaving RuneLite.
+
+ ## Features  
+ **Recommendations**: The plugin automatically fetches and displays recommended gear setups from the OSRS Wiki.
+**Bank Filtering**: This plugin filters items in your bank to show you only the best gear based on your chosen boss or activity.
+**Easy Navigation**: The intuitive interface allows easy searching and selecting bosses and activities. 
```

Comment:
Should put newlines in between these so that they aren't collapsed to one line. Thanks markdown.